### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by gkoz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,13 @@
...
+LGPL-licensed docs for Gtk-rs crates.
+
+The plan is to generate docs from [GIR](https://github.com/gtk-rs/gir-files)
+first then fix and expand them manually.
+
+We're currently in the first phase: no manual corrections while we keep
+improving the [generator](https://github.com/gtk-rs/gir) and regenerating
+these.
+
+The above does not apply to `cairo.md`: cairo doesn't have GIR definitions,
```
Comment (GuillaumeGomez) [inline README.md:12]: I'm wondering: since you linked every other resources, why not cairo?

Comment (gkoz) [inline README.md:12]: Where would the link point to?

Comment (GuillaumeGomez) [inline README.md:12]: To the repo?

Comment (gkoz) [inline README.md:12]: What for? This isn't about the repo and nothing in the repo can help this problem.

Comment (GuillaumeGomez) [inline README.md:12]: That was really curiosity from myself. You answered the question with your previous comment. :)

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**